C-Plus-Plus/other/majority_vote_algorithm.cpp

```diff
@@ -0,0 +1,52 @@
+// The Boyer–Moore majority vote algorithm in C++
```

Comment:
@rutujadhanawade Add a little bit description of the algorithm.

Reply:
Done :+1: @iamrajiv